src/arviz_plots/style.py

```diff
@@ -13,16 +13,79 @@ def use(name):
     -----
     There are some backends where default styles/templates are not supported.
     """
+    ok = False
+
     try:
         import matplotlib.pyplot as plt
 
-        plt.style.use(name)
+        if name in plt.style.available:
+            plt.style.use(name)
+            ok = True
     except ImportError:
         pass
 
     try:
         import plotly.io as pio
 
-        pio.templates.default = name
+        if name in pio.templates:
+            pio.templates.default = name
+            ok = True
+    except ImportError:
+        pass
+
+    if not ok:
+        raise ValueError(f"Style {name} not found.")
+
+
+def available():
```

Comment:
How about returning a dictionary {backend: list of style names} instead of printing? It could be printed later on but would allow checking if a style is available programatically